dashx/src/main/java/com/dashx/sdk/Constants.kt

```diff
@@ -7,9 +7,11 @@ const val PACKAGE_NAME = "com.dashx.sdk"
 const val DEFAULT_INSTANCE = "default"
 
 const val SHARED_PREFERENCES_PREFIX = PACKAGE_NAME
-const val SHARED_PREFERENCES_KEY_BUILD = "$SHARED_PREFERENCES_PREFIX.$DEFAULT_INSTANCE.build"
-const val SHARED_PREFERENCES_KEY_DEVICE_TOKEN = "$SHARED_PREFERENCES_PREFIX.$DEFAULT_INSTANCE.device_token"
+const val SHARED_PREFERENCES_KEY_ACCOUNT_UID = "$SHARED_PREFERENCES_PREFIX.$DEFAULT_INSTANCE.account_uid"
 const val SHARED_PREFERENCES_KEY_ANONYMOUS_UID = "$SHARED_PREFERENCES_PREFIX.$DEFAULT_INSTANCE.anonymous_uid"
```

Comment:
Rename to `account_anonymous_uid` & const to `SHARED_PREFERENCES_KEY_ACCOUNT_ANONYMOUS_UID`